### DISCUSSION TITLE
Filtering by the number of occurences

### DISCUSSION THREAD

**Discussion by darco98** [egeulgen/pathfindR#134] [Q&A] [answered]
Dear all,

We are performing pathway enrichment analysis using pathfindR and we have a doubt. 

If we are doing 100 iterations, do you recommend to filter those pathways that have a low percentage of occurrences (let's say filter out those with less than 10 occurrences) or all the pathways are equally valid? 

Aside from this, how do you determine which is the best number of iterations?

Thank you,
Dani

**Comment by egeulgen** [comment, marked as answer]
Hello,

I would say all enriched pathways/gene sets are equally plausible. As for the number of iterations, because no big variability is expected with the greedy algorithm (default), 10 iterations should suffice. There's unfortunately no suggestions I can give to determine the "best" number of iterations (as there is no ground truth in most cases.

Hope this helps,
-E

**Reply by darco98** [reply]
Thank you for the help :) 
Dani
